tests/rules/name.js

```diff
@@ -30,7 +28,21 @@ var invalidClass = [
     code: code,
     args: [ 1, 'class', 'method' ],
     ecmaFeatures: { classes: true },
-    errors: [{ message: klass }]
+    errors: [{ message: 'class name should start with upper case.' }]
+  };
+});
+
+var missingClass = [
+  'class {' +
+  '}          ',
+  'class extends Bar {' +
+  '}          '
+].map(function(code) {
+  return {
+    code: code,
+    args: [ 1, 'class', 'name-required' ],
+    ecmaFeatures: { classes: true },
+    errors: [{ message: 'class name is required.' }]
   };
```

Comment:
These tests actually fail because of the current state of the (default) esprima parser, which doesn't recognise anonymous classes. 
Babel should work but I just couldn't set it (tried to add property settings: { parser: 'babel-eslint' }).
Any help appreciated